webapp/apps/dynamic/tests/test_views.py

```diff
@@ -15,13 +14,19 @@
 START_YEAR = '2016'
 
 
-class DynamicViewsTests(TestCase):
+class DynamicViewsTests(object):
     ''' Test the views of this app. '''
 
     def setUp(self):
         # Every test needs a client.
         self.client = Client()
 
+    def assertEqual(self, a, b):
+        assert (a == b)
+
+    def assertTrue(self, a):
+        assert a
+
```

Comment:
@lucassz what do you think about dropping the `assert*` methods? When I removed the `TestCase` inheritance from some of the other test classes, I dropped these methods because I think it's easier to just read `a==b` instead of `self.assertEqual(a, b)`.

Reply:
Yes, I think that is a better idea. I was trying to minimize changes to the code, but will make the change.